.travis.yml

```diff
@@ -1,9 +1,18 @@
 language: scala
-script: sbt test scripted
+
+env:
+  matrix:
+    - TRAVIS_SBT_VERSION=0.13.16
+    - TRAVIS_SBT_VERSION=1.0.0-RC3
+
+script: sbt ^^$TRAVIS_SBT_VERSION validate
```

Comment:
Remove all these, we don't want to cross build `sbt-web-build-base` itself yet. It should just be built for sbt 0.13.16. The settings that it provides in `SbtWebBuildBase` should contain the cross build configuration.